static/src/javascripts/projects/commercial/modules/third-party-tags.js

```diff
@@ -0,0 +1,126 @@
+
+// third-party-tags follow an API pattern with their exports, FlowTyped as
+// 'Service'. To avoid naming collisions, namespace imports are used here.
+/* eslint-disable import/no-namespace */
+import * as imrWorldwide from 'commercial/modules/third-party-tags/imr-worldwide';
+import * as imrWorldwideLegacy from 'commercial/modules/third-party-tags/imr-worldwide-legacy';
+import * as remarketing from 'commercial/modules/third-party-tags/remarketing';
+import * as simpleReach from 'commercial/modules/third-party-tags/simple-reach';
+import * as tourismAustralia from 'commercial/modules/third-party-tags/tourism-australia';
+import * as krux from 'commercial/modules/third-party-tags/krux';
+import * as outbrain from 'commercial/modules/third-party-tags/outbrain';
+import * as plista from 'commercial/modules/third-party-tags/plista';
+
+type Service = {
+    shouldRun: boolean,
+    url: string,
+    onLoad?: () => any,
+    useImage?: boolean,
+};
+
+const isLuckyBastard = (): boolean =>
+    testCanBeRun(PaidContentVsOutbrain2) &&
+    getTestVariantId(PaidContentVsOutbrain2.id) === 'paid-content';
+
+const loadExternalContentWidget = (): void => {
+    const externalTpl = template(externalContentContainerStr);
+
+    const findAnchor = (): Promise<any> => {
+        const selector = !(config.page.seriesId || config.page.blogIds)
+            ? '.js-related, .js-outbrain-anchor'
+            : '.js-outbrain-anchor';
+        return Promise.resolve(document.querySelector(selector));
+    };
+
+    const renderWidget = (widgetType, init): void => {
+        findAnchor()
+            .then(anchorNode =>
+                fastdom.write(() => {
+                    $(anchorNode).after(
+                        externalTpl({
+                            widgetType,
+                        })
+                    );
+                })
+            )
+            .then(init);
+    };
+
+    const shouldServePlista: boolean =
+        config.switches.plistaForOutbrainAu &&
+        config.page.edition.toLowerCase() === 'au';
+
+    if (shouldServePlista) {
+        renderWidget('plista', plista.default.init);
+    } else {
+        renderWidget('outbrain', outbrain.init);
+    }
+};
+
+const insertScripts = (services: Array<Service>): void => {
+    const ref = document.scripts[0];
+    const frag = document.createDocumentFragment();
+    while (services.length) {
+        const service = services.shift();
+        if (service.useImage) {
+            new Image().src = service.url;
+        } else {
+            const script = document.createElement('script');
+            script.src = service.url;
+            script.onload = service.onLoad;
+            frag.appendChild(script);
+        }
+    }
+    if (ref.parentNode) {
+        ref.parentNode.insertBefore(frag, ref);
+    }
+};
+
+const loadOther = (): void => {
+    const services = [
+        audienceSciencePql,
```

Comment:
Modules are still there for now 🙃 

May be a later PR to delete it all from frontend